### ISSUE TITLE
Hide Button doesn't work on Runelite launch

### ISSUE THREAD

**Issue by etc597** [Hablapatabla/ImplingFinder#9]
So I have hide button set because I want to contribute data and I don't really care about hunting implings. However, the button is visible on launch even if that option is set (have set it, closed runelite, opened runelite, button is showing yet the option is still saved as being set). Not really sure what's going on but it would be nice to have this addressed.

**Comment by Hablapatabla** [comment]
hi! thanks for letting me know, ill look into this tomorrow, should be an easy fix

**Comment by Hablapatabla** [comment]
hey so im not seeing the issue on my side. im gonna ask a few others and see if it can duplicate. you've restarted runelite, as well as impling finder, and when you toggle the hide button on and off theres no movement in your icon panel on the right?

**Comment by etc597** [comment]
The toggle works for me but when I close and reopen runelite, the button in the side panel reappears despite having the option set. I’ll try again and see. I am logged into runelite so my stuff is synced between my computers so maybe that has something to do with it?

**Comment by etc597** [comment]
Yeah just checked again, it’s still occurring. I can get a video if you’d like?

**Comment by Hablapatabla** [comment]
hey buddy, sorry for leaving this dead. is it still an issue for you? i saw nothing in my logs and other people seem to be working fine so lmk and if so i may talk to runelite people about it

**Comment by etc597** [comment]
Still an issue:
![image](https://github.com/Hablapatabla/ImplingFinder/assets/16726388/fa7e5bc4-67e5-48b7-b9e2-ef0f070d76ae)

**Comment by etc597** [comment]
It gets rid of the button initially but upon re-opening runelite it appears again
